stable/stolon/README.md

```diff
@@ -0,0 +1,64 @@
+# Stolon Helm Chart
+
+* Installs [Stolon](https://github.com/sorintlab/stolon) (HA PostgreSQL cluster)
+* Inspired by [this](https://github.com/lwolf/stolon-chart) and [stolon examples](https://github.com/sorintlab/stolon/tree/master/examples/kubernetes/statefulset)
+
+## TL;DR;
+
+```console
+$ helm install stable/stolon
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/stolon
+```
+
+## Backend
+
+Kubernetes is the default store backend. `consul`, `etcdv2` or `etcdv3` can also be used as the store backend.
+
+## Configuration
+
+| Parameter                               | Description                                    | Default                                                      |
+| --------------------------------------- | ---------------------------------------------- | ------------------------------------------------------------ |
+| `image.repository`                      | `stolon` image repository                      | `sorintlab/stolon`                                           |
+| `image.tag`                             | `stolon` image tag                             | `master-pg10`                                               |
```

Comment:
You changed this from 0.10. I'd prefer to have a versioned image instead of a "rolling-update" one. Any reason why you switched?

Reply:
We require stable version. Any idea when the next release is due?

Reply:
@unguiculus @timstoop  I was incorrect about needing to use the master image version.  Reverted to stable version and everything looks to be working.